tyk-data-plane/README.md

```diff
@@ -0,0 +1,174 @@
+## Tyk Data Plane
+
+tyk-data-plane provides the default deployment of a Tyk-Data-Plane. It includes the Tyk Gateway, an open source Enterprise API Gateway (
+supporting REST, GraphQL, TCP and gRPC protocols) and Tyk Pump, an analytics purger that moves the data generated by your Tyk nodes to any back-end. Furthermore 
+it has all the required modifications to easily connect to tyk-control-plane using MDCB.
```

Comment:
Not only MDCB, this should be able to connect to control plane on Tyk Cloud, a remote cluster (MDCB) or same cluster (Self-managed control plane)